exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/EntityClass.kt

```diff
@@ -954,12 +956,79 @@ abstract class EntityClass<ID : Comparable<ID>, out T : Entity<ID>>(
         }
     }
 
+    internal fun warmUpAllReferences(
```

Comment:
Probably better to name `warmUpCompositeIdReferences`

I read this code first time, and I haven't expected that additionally to the method `warmUpAllReferences` there is another one `warmUpReferences`. That's definitely minor.